Editor/CesiumEditorWindow.cs

```diff
@@ -146,12 +148,16 @@ void DrawCesiumToolbar()
                 "Upload a tileset to Cesium ion to process it for efficient streaming to Cesium for Unity",
                 true,
                 this.UploadToIon);
+
+            EditorGUI.BeginDisabledGroup(!this._isAuthenticationRequired);
```

Comment:
I haven't been able to test the UI, so I don't know. When this button is disabled, is it visually affected, e.g. greyed out? If so, great! If not, I think that it will be a little confusing if the user tries to click it but finds it can't open.

Reply:
Yep, it's greyed out!